client/src/components/FooterButton/FooterButton.jsx

```diff
@@ -13,36 +12,33 @@ const Container = styled.div`
 const Icon = styled.img`
   height: ${(props) => props.size};
   width: ${(props) => props.size};
+  background-color: #667d7c;
+  border-radius: 50%;
+  border: 0px;
 `;
 
 const Text = styled.p`
   margin: 1rem 0 0;
   color: ${colors.white};
 `;
 
-function AddHeroButton({ handleClick }) {
+function FooterButton({ icon, handleClick, children }) {
   return (
     <Container>
-      <Icon
-        size="50px"
-        src={addHeroIcon}
-        onClick={() => {
-          handleClick("addHero");
-        }}
-      />
-      <Text
-        onClick={() => {
-          handleClick("addHero");
-        }}
-      >
-        Add a hero
-      </Text>
+      <Icon size="50px" src={icon} onClick={handleClick} />
+      <Text onClick={handleClick}>{children}</Text>
     </Container>
   );
 }
 
-AddHeroButton.propTypes = {
+FooterButton.propTypes = {
+  icon: PropTypes.string,
   handleClick: PropTypes.func.isRequired,
+  children: PropTypes.string.isRequired,
 };
 
-export default AddHeroButton;
+FooterButton.defaultProps = {
+  icon: "default.svg",
```

Comment:
This can also be imported directly in js file, instead of placing directly in public folder.